docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -160,6 +160,67 @@ There are a variety of types supported via the <PyObject object="MetadataValue"
 
 ---
 
-## Executing checks via the UI
+## Executing checks
+
+### Via the UI
 
 Materializing an asset from the UI will also execute any checks that are defined for that asset. You can also execute checks without materializing the asset from the Checks tab of the asset’s detail page.
+
+### Via sensors and schedules
+
+You can use <PyObject object="define_asset_job" /> to define jobs that include sets of both assets and checks, and then execute those jobs via [sensors](/concepts/partitions-schedules-sensors/sensors) and [schedules](/concepts/partitions-schedules-sensors/schedules).
+
+```python file=/concepts/assets/asset_checks/jobs.py
+from dagster import (
+    AssetSelection,
+    Definitions,
+    ScheduleDefinition,
+    asset,
+    asset_check,
+    define_asset_job,
+)
+
+
+@asset
+def my_asset():
+    ...
+
+
+@asset_check(asset=my_asset)
+def check_1():
+    ...
+
+
+@asset_check(asset=my_asset)
+def check_2():
+    ...
+
+
+# includes my_asset and both checks
+my_job = define_asset_job("my_job", selection=AssetSelection.assets(my_asset))
+
+
+# includes only my_asset
+my_asset_only_job = define_asset_job(
+    "my_asset_only_job",
+    selection=AssetSelection.assets(my_asset) - AssetSelection.all_asset_checks(),
```

Comment:
I thought we decided we were going to use `without_checks` instead of the difference operator?